### ISSUE TITLE
implement additive genetic model?

### ISSUE THREAD

**Issue by freblisna** [nhgritctran/PheTK#19]
Thanks for a great and easy-to-use package! It worked pretty seamless in All of Us's new platform, once I overcame the initial permissions error as described in #17.

To follow up on #7 , it is currently possible to store genotype values as case / control (0 or 1) and I can modify the dictionary to give me dosages (0: 0/0, 1: 0/1, 2: 1/1). However, the only models available in "PheWAS" are logit/cox/firth_logit/firth_cox. Is there a recommended method available for modeling dosages? I have a gwas variant that I don't want to assume is recessive or dominant and generally I assume it's additive. I imagine a linear model, rather than logits or cox?

**Comment by nhgritctran** [comment]
In standard PheWAS as well as PheTK, each phecode regression is: phecode ~ variable of interest + covariates. 

The dependent variable phecode is a binary variable and therefore logistic/cox regressions are appropriate methods. Your variable of interest can be of any numerical type, though the specific type would dictate how you interpret the results.

**Comment by freblisna** [comment]
Oh, I see. This is my mis-understanding. Genotype isn't the response variable, it's a predictor of phecode. That makes sense.

Thanks for clarifying and for the fast response!
